.github/workflows/master-to-relrc.yml

```diff
@@ -0,0 +1,24 @@
+name: Create rel/rc pull request
+on:
+  push:
+    branches:
+      - master
+jobs:
+  relrc-promotion:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: rel/rc
+      - name: Merge master to rel/rc
+        run: |
+          git fetch origin master:master
+          git reset --hard master
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: format('chore', ':', 'merge master to rel/rc')
```

Comment:
Since there's no changes made on line #14 (run), `commit-message` won't take any effect in this case. That's why they don't include `commit-message` in the example of CPR.

Reply:
Just a couple of screenshots on how it looks like in my private test repo. The `commit-message` doesn't have any effect even though we specify it as `format('chore', ':', 'merge master to rel/rc')`.

<img width="803" alt="image" src="https://user-images.githubusercontent.com/103960489/170790929-9e706d8e-1634-4e0d-b3cb-fb945ce1afba.png">
<img width="803" alt="image" src="https://user-images.githubusercontent.com/103960489/170791022-f7c5f1fd-3ebc-4624-a4bc-1db513dc52ae.png">

Reply:
Thanks, going to remove this line.